### PR TITLE
added new content docu-1173

### PR DESCRIPTION
**Pull request by fi3ld**
https://jira.wandisco.com/browse/DOCU-1173

### CODE REVIEW

File: docs/configuration-ldm.md [hunk 1]
```diff
@@ -180,7 +186,34 @@ Adjust notification properties in the `application.properties` file:
...
+| Name | Details |
+| --- | --- |
+| JVM_MAX_MEM="value" | Specifies the maximum size of the heap that can be used by the Java virtual machine (JVM). Use k, m, or g (case insensitive) for KiB, MiB, or GiB. |
+| JVM_MIN_MEM="value" | Specifies the minimum size of the heap that can be used by the Java virtual machine (JVM). Use k, m, or g (case insensitive) for KiB, MiB, or GiB. |
+| JVM_GC="value" | Specifies the type of garbage collecter. |
+| JVM_G1HeapRegionSize="value" | Specifies the heap region size. Minimum 1 maximum 32 Mb with values that are always a power of 2. |
+| JVM_GC_LOG_NAME="value” | Specifies the GC log name. |
+| JVM_GC_LOG_SIZE="value" | Specifies the max GC log size. |
+| JVM_GC_NUM_LOGS="value" | Specifies the max number of GC logs. |
+| HTTP_MAX_CONNECTIONS="value" | Specifies the maximum number of keep-alive connections that are maintatined. The possible range is zero to 32768. |
```
Comment (AshVic) [inline docs/configuration-ldm.md:204]: Typo: maintained

**Review by lucasdigital — APPROVED** [review]
Looks good. Just one comment.

**Review by AshVic — CHANGES_REQUESTED** [review]
General comments:

Usually we should write a product name or trademarked name out in full at the beginning and then after first instance use the abbreviation. eg. Transport Layer Security (TSL) first time, and thereafter, just TSL.

Also I would be consistent when using title and sentence case in headers. There is a mix of both for level 3 headers.

One small typ further down to correct.

File: docs/configuration-ldm.md (hunk 1 above)
Comment (lucasdigital) [reply docs/configuration-ldm.md:204]: ooh missed that!

**Comment by fi3ld** [comment]
I need to change ### Directory structure to ## Directory structure.

**Comment by fi3ld** [comment]
> Line 215. We've stopped using Latin abbreviations, so I would change eg into (For example: LDM_...)
> Also, add a full stop to the line for consistency -- or remove the ones included on the other lines.

Done

**Comment by fi3ld** [comment]
> I need to change ### Directory structure to ## Directory structure.

Done

File: docs/configuration-ldm.md (hunk 1 above)
Comment (fi3ld) [reply docs/configuration-ldm.md:204]: Done

File: docs/configuration-ldm.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ When basic authentication is enabled on LiveData Migrator, [update the LiveData
 | `security.basic.user` | Required when `security.type=basic`. <br/>The username that needs to be provided by a REST client to gain access to a secured REST API, for example: `admin`<br/><br/>[If basic authentication is enabled or will be enabled on the HiveMigrator REST API](./configuration-hvm.md#basic-authentication), use the same username for LiveData Migrator and HiveMigrator.<br/><br/>**Default value**: (none)<br/>**Allowed values**: Any string that defines a username (no whitespace) |
 | `security.basic.password` | Required when `security.type=basic`. <br/>A [bcrypt-encrypted](https://www.browserling.com/tools/bcrypt) representation of the password that needs to be provided using HTTP basic authentication to access the REST API, for example:<br/>`{bcrypt}$2a$10$mQXFoGAdLryWcZLjSP31Q.5cSgtoCPO3ernnsK4F6/gva8lyn1qgu`<br/><br/>The `{bcrypt}` prefix must be included before the encrypted password string as shown in the example above.<br/><br/>**Default value**: (none)<br/>**Allowed values**: A valid bcrypt-encrypted string |
 
-#### Connecting to LiveData Migrator with basic authentication
+#### Connecting To LiveData Migrator With Basic Authentication
```
Comment (AshVic) [inline docs/configuration-ldm.md:81]: Use lower case for prepositions in titles/headers: "to"

File: docs/configuration-ldm.md [hunk 3]
```diff
@@ -201,8 +201,8 @@ Adjust notification properties in the `application.properties` file:
 | JVM_GC_LOG_NAME="value” | Specifies the GC log name. |
 | JVM_GC_LOG_SIZE="value" | Specifies the max GC log size. |
 | JVM_GC_NUM_LOGS="value" | Specifies the max number of GC logs. |
-| HTTP_MAX_CONNECTIONS="value" | Specifies the maximum number of keep-alive connections that are maintatined. The possible range is zero to 32768. |
-| LDM_EXTRA_JVM_ARGS=”value” | Add extra Java arguments (eg LDM_EXTRA_JVM_ARGS=“-Dhttps.proxyHost=sub.proxy.att.com -Dhttps.proxyPort=8080”) |
+| HTTP_MAX_CONNECTIONS="value" | Specifies the maximum number of keep-alive connections that are maintained. The possible range is zero to 32768. |
+| LDM_EXTRA_JVM_ARGS=”value” | Add extra Java arguments (for example LDM_EXTRA_JVM_ARGS=“-Dhttps.proxyHost=sub.proxy.att.com -Dhttps.proxyPort=8080”). |
```
Comment (AshVic) [inline docs/configuration-ldm.md:205]: Comma after "For example,"

**Review by AshVic — CHANGES_REQUESTED** [review]
Minor changes otherwise LGTM

**Pull request merged**